### DISCUSSION TITLE
Document sample certificate-generating script

### DISCUSSION THREAD

**Discussion by thehowl** [smart-on-fhir/health-cards#226] [Show and tell]
While working on the implementation of SMART Health Cards for my client, due to the requirements for the certificates I came up with a bash script to generate the necessary certificates and jwks.json file mentioned in the spec: https://gist.github.com/thehowl/2d57263fc0c700f75c743e14106715e4

I'm posting this here as this might be a useful reference for anyone looking to implement the spec, but I was also wondering whether the spec should also include this script (or something different which might have been more thoroughly reviewed) so that the JWK generation "step" in the implementation becomes much more trivial?

**Comment by jmandel** [comment]
Thanks for the comment here! I'll note that we have some related examples in https://github.com/smart-on-fhir/health-cards/tree/main/generate-examples but will migrate this thread to discussions so it's available for others who are browsing.
